website/docs/r/kusto_cluster.html.markdown

```diff
@@ -82,7 +82,9 @@ The following arguments are supported:
 
 * `zones` - (Optional) Specifies a list of Availability Zones in which this Kusto Cluster should be located. Changing this forces a new Kusto Cluster to be created.
 
-* `engine` - (Optional). The engine type that should be used. Possible values are `V2` and `V3`. Defaults to `V2`.
+* `engine` - (Optional). The engine type that should be used. Possible values are `V2` and `V3`. Defaults to `V2`. Changing this forces a new Kusto Cluster to be created.
+
+~> **NOTE:** In v4.0 and later version, default engine type will be `V3`. 
```

Comment:
quoting
```suggestion
~> **NOTE:** In `v4.0` and later version, default engine type will be `V3`. 
```

also what v4 are you referring to?

Reply:
I am refering to the azurerm provider here, let me change the discription